src/main/java/com/jpa/entity/AbstractIdentifiableObject.java

```diff
@@ -0,0 +1,20 @@
+package com.jpa.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+
+
+@MappedSuperclass
+public abstract class AbstractIdentifiableObject {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Getter
+    @Setter
+    private int id;
```

Comment:
https://stackoverflow.com/questions/7506802/using-wrapper-integer-class-or-int-primitive-in-hibernate-mapping